backend/e4h-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -116,6 +116,104 @@ async def _save_upload_to_temp_file(upload_file: UploadFile, suffix: str = ".xls
+                df.at[index, 'error'] = f'Exception during bulk create: {str(exc)}'
+            continue
+
+        if create_resp.status_code in (200, 201):
+            for row_idx in creation_meta:
+                df.at[row_idx, 'status'] = 'success'
+                df.at[row_idx, 'error'] = ''
+        elif create_resp.status_code == 400:
+            try:
+                error_data = create_resp.json()
+                error_message = error_data.get('Errors', [{}])[0].get('message', 'Unknown error')
+            except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)